pkgs/development/libraries/openssl/default.nix

```diff
@@ -154,7 +153,7 @@ let
     postInstall =
     (if static then ''
       # OPENSSLDIR has a reference to self
-      ${removeReferencesTo}/bin/remove-references-to -t $out $out/lib/*.a
+      ${buildPackages.removeReferencesTo}/bin/remove-references-to -t $out $out/lib/*.a
```

Comment:
```suggestion
      remove-references-to -t $out $out/lib/*.a
```
Why can't we add it to nativeBuildInputs?

Reply:
#192084

Reply:
Sure, either way is cool with me. A couple of lines down `${buildPackages....}` was used for something else so I just did it this way to keep it consistent.

Reply:
In a replace we cannot really use nativeBuildInputs.